spec/spec_helper.rb

```diff
@@ -0,0 +1,17 @@
+RSpec.configure do |config|
+    config.order = :default
+end
+
+# Define the variabe board
+cell_1 = "X"
+cell_2 = " "
+cell_3 = "O"
+
+cell_4 = "X"
+cell_5 = "O"
+cell_6 = "X"
+
+cell_7 = "X"
+cell_8 = "O"
+cell_9 = "X"
```

Comment:
Looks like you had plan to use these variables before, but since I don't see reference to them anywhere. Could they be removed, too?